package.json

```diff
@@ -37,7 +37,7 @@
     }
   },
   "devDependencies": {
-    "electron": "2.0.7",
+    "electron": "^2.0.8",
```

Comment:
I opted for `^` semver to track 2.x.y, but happy to consider more pessimistic pinning if others have strong opinions about it.

Reply:
@jonsmorrow You said "We pin to specific versions" when we chatted about this vulnerability. Do you have a strong opinion to continue specific pinning?

Reply:
We can try it. Most examples suggest hard pinning we can always go back if need be.